articles/recommender-system-explained/index.md

```diff
@@ -0,0 +1,89 @@
+
+#### Multi-criteria recommender systems
+These systems provide recommendations based on multiple criteria relating to preference information. The system analyzes the preference information of a user for an explored item using various criteria, rather than a single criterion. It then forecasts a rank for unexplored products and provides relevant suggestions to the user. 
+
+#### Mobile recommender systems
+These systems use smartphone devices to offer personalized recommendations. The accuracy of these systems depends on the context, privacy, and recommendation approach. A good example of a mobile recommender system is the one used by taxi companies such as Lyft, DiDi, and Uber. In such a case, the system uses the GPS information of routes used by drivers to recommend appropriate pick-up points. The GPS information used includes the operational status, location, and time stamps. 
+
+#### Hybrid recommender systems
+Hybrid recommender systems combine content-based filtering and collaborative filtering approaches. These systems help in overcoming some of the limitations of pure recommender systems such as sparsity problems and cold start. Netflix is an example of a company that employs hybrid recommender systems.
+
+### How recommender systems work
+Recommender systems work by using users' data to generate suggestions. The following image provides an overview of how recommender systems work. 
+
+![How Recommender Systems Work](/engineering-education/recommender-system-explained/how-recommender-systems-work.jpg)
+
+[Image Source: Imaginet](https://www.imaginet.com/wp/wp-content/uploads/2020/07/process-1.png)
+
+The first step involves data collection to gather the ratings, tastes, preferences, and online behavior of customers. The main sources of data include explicit ratings and implicit ratings. Information relating to product similarity and user similarity is also used to provide the basis for recommendation. 
+
+The collected data is then stored and used in the training phase. This phase involves conducting statistical analysis and applying mathematical algorithms to learn the existing patterns. The last stage involves suggesting items to the online user. In this stage, a model is deployed into a container that generates a REST API. The web application uses the customer data to send queries to the recommendation system. The system then provides the recommendation and displays it on the customer’s page. 
+
+The algorithms used and the recommendation approach depend on the type of recommender system. For example, a collaborative filtering system analyzes data and establishes two users that have bought the same product (s). If one of the users buys another product, the system will recommend it to the other user because they have similar preferences. The following image shows how this type of system works. 
```

Comment:
```suggestion
The algorithms used and the recommendation approach depend on the type of recommender system. For example, a collaborative filtering system analyzes data and identifies two users that have bought the same product (s). If one of the users buys another product, the system will recommend it to the other user because they have similar preferences. The following image shows how this type of system works.